tests/test_store.py

```diff
@@ -1,6 +1,6 @@
 import os.path
 import tempfile
-import unittest.mock as mock
+from unittest import mock
```

Comment:
Is this a Pylint suggestion? If not, I almost think the previous form is clearer... to avoid mixing `from` and `import` for imports, if not for anything else.

Reply:
Yep, it's a pylint suggestion. Partially agree, but the complaint was that things should not be aliased to what they are already called, which I also think is reasonable.